docs/howto/migrate-s3-to-s3.md

```diff
@@ -0,0 +1,45 @@
+# Migrate from one S3 backend to another
+
+This can be from one Riak CS cluster to another,
+or from a Riak CS cluster to Amazon S3, for example.
+
+## Send new writes to the new S3 endpoint
+
+1. If the new endpoint is a riakcs cluster,
+   add `[riakcs_new]` hosts (of new riak cluster) to inventory.
+   (Otherwise skip this step.)
+2. Add `localsettings.BLOB_DB_MIGRATING_FROM_S3_TO_S3: True`
+   in `environments/<env>/public.yml`
+3. Add/update settings in `environments/<env>/vault.yml`
+  - `secrets.RIAKCS_S3_ACCESS_KEY` (new cluster)
+  - `secrets.RIAKCS_S3_SECRET_KEY` (new cluster)
+  - `secrets.OLD_S3_BLOB_DB_ACCESS_KEY` (old cluster)
+  - `secrets.OLD_S3_BLOB_DB_SECRET_KEY` (old cluster)
```

Comment:
nit: I think items in this list should be indented one more space to make it a sub-list of step # 3.